packages/ui-tree-browser/src/TreeBrowser/index.js

```diff
@@ -395,4 +399,4 @@ class TreeBrowser extends Component {
 }
 
 export default TreeBrowser
-export { TreeBrowser }
+export { TreeBrowser, TreeNode }
```

Comment:
Why does TreeNode need to be exported here?

Reply:
from https://github.com/instructure/instructure-ui/blob/master/docs/getting-started/api-guidelines.md --

> Child components that can't be used on their own should be exported as static properties on the parent component (e.g. Table.Row, Menu.Item), so that only one import is required to render the component (e.g. import Table from '@instructure/ui-tables').

I'm not sure if docs export is needed (seems redundant with this line) but I couldnt get the example in the TreeBrowser/README.md to work otherwise

Reply:
For the Table.Row example that is called out, this is out its exported as a static property:

https://github.com/instructure/instructure-ui/blob/910d3630c55716835b2dadcca228a5a197175919/packages/ui-table/src/Table/index.js#L97 inside of Table/index.js

Reply:
oh, I can't read.